EfEnumToLookupTests/Tests/TestStuff.cs

```diff
@@ -19,8 +19,11 @@ public void SetUp()
 			// Using setup rather than teardown to make it easier to inspect the database after running a test.
 			using (var context = new MagicContext())
 			{
-				context.Database.Delete();
-			}
+                if(context.Database.Exists())
```

Comment:
Did you have problems with this bit? just wondering what caused you to add the `if` around it. Worked on my machine as it was. This should imho be a separate commit with an explanation, though no big deal really and it wouldn't stop me accepting.